packages/server/src/data/models/model-postgres.js

```diff
@@ -829,6 +845,85 @@ class ModelPostgres extends Model {
     )`;
   }
 
+
+  /**
+   * @todo: see this._publicAndJoinedChannels()
+   * @protected
+   * @param {Defs.ChannelID} channelId
+   * @param {string} name
+   * @return {Sequelize.literal}
+   */
+  _agentsIdsLiteral(channelId, name = '') {
+    return this.literal(this._agentsIds(channelId));
+  }
+
+  /**
+   * @todo: Use ORM opportunities to join tables.
+   * @todo: No, do not this ORM for this.
+   * @protected
+   * @param {Defs.ChannelID} channelId
+   * @param {string} name
+   * @return {string}
+   */
+  _agentsIds(channelId, name = '') {
+    assert(!!channelId, 'Channel ID is required.');
+    return `(
+      /* Agent Ids (${name}) */
+      SELECT "ChannelMemberships"."agentId" FROM "ChannelMemberships"
+      WHERE "ChannelMemberships"."channelId" = '${channelId}'
+    )`;
+  }
+
+  /**
+   * @param {string} [name]
+   * @return {string}
+   */
+  _verifiedMeasurementsLiteral(name = '') {
+    return this.literal(this._verifiedMeasurements(name));
+  }
+
+  /**
+   * @protected
+   * @param {string} name
+   * @return {string}
+   */
+  _verifiedMeasurements(name = '') {
+    return `(
+      /* Verified Channel Memberships (${name}) */
+      SELECT "ChannelMemberships"."agentId"
+      FROM "ChannelMemberships"
+      LEFT JOIN "Channels" ON "Channels"."id" = "ChannelMemberships"."channelId"
+      WHERE "ChannelMemberships"."isVerified" = TRUE
+      OR "Channels"."requireVerification" != TRUE
```

Comment:
Why are you using direct SQL instead of sequelize here?

Reply:
Because Sequelize has its own edges of usage. And for this purpose, this is not a good library. Sequelize is a good library for small and simple application. In the future, I would recommend to remove Sequelize at all either to reduce its usage.

Reply:
Makes sense. Are there other libraries you'd recommend instead, or just use raw SQL much more?

Reply:
It depends on issues. Do not think about SQL / Sequelize much. SQL is metalanguage between application and database. First of all thing about application architecture: data layers (restrictions, filters), middleware, validators, authorizes, graphql interface, resolvers, data definitions, sync flows.

I do not imagine how to explain this better.  So for this step of application growing the Sequelize is ok. I think the most use cases are coded in the code-base. Now Sequelize is a more DB driver and is used under Data / Models layers.